omarchy-plugin/lib/pomo_link/rest.py

```
@@ -57,16 +51,16 @@ def request(self, method, path, body=None, timeout=None, host=None, port=None, t
             method=method,
         )
         try:
-            with self.opener.open(req, timeout=timeout) as resp:
+            with urllib.request.urlopen(req, timeout=timeout) as resp:
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)